e2e/jest.test-setup.ts

```diff
@@ -17,14 +18,16 @@ beforeEach(async () => {
   page.setDefaultNavigationTimeout(60000); // Puppeteer default timeout is 30 seconds.
   await page.setRequestInterception(true);
 
-  const getTitle = async () => {
+  const getPageTitle = async () => {
     return await page.$eval('title', title => {
       return title.textContent;
-    }).catch(() =>  {return 'getTitle() func failed'});
+    }).catch(() =>  {return 'getPageTitle() func failed'});
   }
 
-  const describeJsHandle = async (jsHandle)  => {
+  const describeJsHandle = async (jsHandle: JSHandle): Promise<string>  => {
     return jsHandle.executionContext().evaluate(obj => {
+      if (obj instanceof Error)
```

Comment:
Please use brackets, also please add a comment - I don't follow this

Reply:
Done.